src/components/Images/Images.module.scss

```diff
@@ -0,0 +1,32 @@
+/* @getethos/design-system/Images.scss
+   ========================================================================== */
+
+// WIP.
```

Comment:
My test caught something! 😃

Monorepo webpack blows up if you use `//` syntax for comments.

Reply:
i'm pushing some fixes rn